astropy/io/fits/tests/test_core.py

```diff
@@ -1278,6 +1285,14 @@ def _make_bzip2_file(self, filename='test0.fits.bz2'):
 
         return bzfile
 
+    def test_simulateonly(self):
+        """Write to None simulates writing."""
+
+        with fits.open(self.data('test0.fits')) as hdul:
+            hdul.writeto(None)
+            hdul[0].writeto(None)
+            hdul[0].header.tofile(None)
```

Comment:
Never knew this feature existed! What is it for?

Reply:
:man_shrugging: